nat-lab/tests/test_dns.py

```diff
@@ -704,13 +677,14 @@ async def test_dns_duplicate_requests_on_multiple_forward_servers() -> None:
         await testing.wait_normal(
             connection_alpha.create_process(
                 ["nslookup", "google.com", DNS_SERVER_ADDRESS]
-            ).execute(),
+            ).execute()
         )
 
         await asyncio.sleep(1)
 
         results = re.findall(
-            r".* IP .* > (?P<dest_ip>\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,5}): .* A\?.*",
+            r".* IP .* > (?P<dest_ip>\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,5}): .*"
+            r" A\?.*",
```

Comment:
Is there an easy way to add some skip/ignore for this line. Feels like that could stay as one line

Reply:
looking into it :eyes:

Reply:
`# fmt: skip` might be a solution, what do you think?

Reply:
If it works I would use it here

Reply:
it does, already added :smile: